rollup.config.js

```diff
@@ -9,12 +14,17 @@ export default [
       { file: pkg.main, format: 'cjs' }
     ],
     plugins: [
+      // Allows node_modules resolution
+      resolve({ extensions }),
+      // Compile TypeScript/JavaScript files
       babel({
+        extensions,
         babelrc: false,
```

Comment:
Didn't change in this PR, just wonder why we don't use the babelrc file, and declare things again in the rollup.config.

Reply:
`.babelrc` is used by test script and it has `rewire` plugin in it, but ideally we should have only one